lib/puppet/functions/postgresql/postgresql_password.rb

```diff
@@ -6,15 +6,25 @@
   #   The clear text `username`
   # @param password
   #   The clear text `password`
+  # @param sensitive
+  #   If the Postgresql-Passwordhash should be of Datatype Sensitive[String]
   #
   # @return [String]
```

Comment:
If you set `return_type` (which I like), you can drop the type here.
```suggestion
  # @return
```

Reply:
Suggestion implemented.